api/package.json

```diff
@@ -20,7 +20,12 @@
   "dependencies": {
     "@filecoin-station/spark-impact-evaluator": "^1.1.1",
     "@glif/filecoin-address": "^3.0.5",
+<<<<<<< feat-log-network-info
```

Comment:
merge conflicts committed

Reply:
Weird, I used GitHub's editor for once and removed it, but maybe I needed to save the page or something before marking as resolved.

Addressed in https://github.com/filecoin-station/spark-api/pull/339/commits/d169b6d253000391c888ed9ff090eea6d3ae0773